rero_mef/agents/gnd/mappings/v7/agents_gnd/gnd-agent-v0.0.1.json

```diff
@@ -17,6 +17,9 @@
       "bf:Agent": {
```

Comment:
Why not just rename this field ? This one should be deleted as it is now redundant.

Reply:
rero-ils is still using this.